assets/js/common/Table/search.js

```diff
@@ -0,0 +1,23 @@
+const canonicalise = (val) => val.normalize().trim().toLowerCase();
+
+const stringContains = (subject, predicate) =>
+  canonicalise(subject).includes(canonicalise(predicate));
+
+const arrayContains = (subject, predicate) =>
```

Comment:
Again, this is not a predicate. Also you don't need the reduce here, I think this is a bit more succinct, what do you think?

```suggestion
const arrayContains = (strings, searchString) =>
  strings
    .map((s) => stringContains(s, searchString))
    .includes(true);
```